scripts/release/release-notes-onprem.sh

```diff
@@ -116,15 +116,32 @@ if [ -n "$IS_RELEASE" ]; then
   echo "Creating ${RELEASE_VERSION} in Jira ${JIRA_PROJECT_ID}"
   payload_version="{\"archived\":false,\"name\":\"${RELEASE_VERSION}\",\"projectId\":${JIRA_PROJECT_ID},\"released\":false,\"description\":\"OnPrem Release for ${RELEASE_VERSION} by $JIRA_USER\"}"
   payload_issue="{\"update\": {\"fixVersions\": [{\"add\": {\"name\": \"${RELEASE_VERSION}\"}}]}}"
+  echo "payload_issue is $payload_issue"
+  echo "payload_version is $payload_version"
+  echo "JIRA_AUTH is $JIRA_AUTH"
+
   response=$(curl -s --request POST \
      --url "$JIRA_URL/rest/api/3/version" \
      --user "$JIRA_AUTH" \
      --header 'Accept: application/json' \
      --header 'Content-Type: application/json' \
      --data "${payload_version}")
+  echo "create version response is $response"
   error_message=$(echo $response | jq '.errorMessages')
   if [ "$error_message" != "null" ]; then
-    echo "Error creating version: $error_message"
+    #
+    # if you need to delete a version, which was created accidentally
+    # curl --request DELETE \
+    #  --url 'https://smartbear.atlassian.net/rest/api/3/version/41330' \ 
+    #  --user "$JIRA_AUTH" \
+    #  --header 'Accept: application/json'
+    #
+    #  replace 41330 with version ID you want to delete
+    #  you can see the version ID in jira UI
```

Comment:
if you are looking at the UI, you may as well delete it there ;)

Reply:
to be honest, i wouldn't want this here, if someone accidentally picks the wrong ID, they will delete an acctual release. I would prefer if they ask for some help if they get into this situation.